examples/auth/solidstart/.env.example

```
@@ -0,0 +1,3 @@
+# Find these values at https://supabase.com/dashboard/project/_/settings/api
+VITE_SUPABASE_URL=https://slmzeiwqohkvgzembhii.supabase.co
+VITE_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InNsbXplaXdxb2hrdmd6ZW1iaGlpIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NjcwOTQzMTQsImV4cCI6MjA4MjY3MDMxNH0.bHDFA0JrqhppjQfomsSN6YYSRA9fCxon5MWHNGd1wds
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SUPABASE_ANON_KEY key should go before the VITE_SUPABASE_URL key

(UnorderedKey)